mods/persistence/datums/wires/wires.dm

```diff
@@ -0,0 +1,5 @@
+//Fix annoyance
```

Comment:
Comment is unclear, probably not necessary.

Reply:
The comment is explained on line 3